packages/ia-userlist-settings/src/ia-user-list-settings.ts

```diff
@@ -40,6 +40,8 @@ export class IAUserListSettings extends LitElement {
 
   private async saveListDetails(event: Event) {
     event.preventDefault();
+    const submitButton = (event.target as HTMLElement)?.querySelector('button#save-list-settings');
```

Comment:
Instead of playing with with DOM, can't we simply use lit state to disable the button after initial click?

Reply:
ideally, that's the route. hope we can fix it next pass